html/controls/lt/lt-sheet.js

```diff
@@ -271,61 +263,81 @@ function prepareFieldingEditor(teamId) {
 
 	function initialize() {
 		var table = $('<table>').appendTo($('#FieldingEditor'));
+		
 		var row = $('<tr>').addClass('Skater').appendTo(table);
-		$('<td>').append($('<select>').attr('id', 'skater').append($('<option>').attr('value', '').text('None/Unknown'))).appendTo(row);
-		var notFieldedField = $('<td>').append($('<button>').attr('id', 'notFielded').button().text('No Skater fielded')).appendTo(row).children('button');
-		notFieldedField.click(function(){notFieldedField.attr('checked', notFieldedField.attr('checked') == null);});
-		var sitFor3Field = $('<td>').append($('<button>').attr('id', 'sitFor3').button().text('Sit out next 3')).appendTo(row).children('button');
-		sitFor3Field.click(function(){sitFor3Field.attr('checked', sitFor3Field.attr('checked') == null);});
+		$('<td>').append($('<select>').attr('id', 'skater').append($('<option>').attr('value', '').text('None/Unknown')).change(function() {
+			WS.Set(fieldingEditor.data('prefix')+'Skater', $(this).val());
+		})).appendTo(row);
+		$('<td>').append($('<button>').attr('id', 'notFielded').button().text('No Skater fielded').click(function() {
+			var check = $(this).attr('checked') == null;
+			$(this).attr('checked', check);
+			WS.Set(fieldingEditor.data('prefix')+'NotFielded', check);
+		})).appendTo(row)
+		$('<td>').append($('<button>').attr('id', 'sitFor3').button().text('Sit out next 3').click(function() {
+			var check = $(this).attr('checked') == null;
+			$(this).attr('checked', check);
+			WS.Set(fieldingEditor.data('prefix')+'SitFor3', check);
+		})).appendTo(row);
+		
+		row = $('<tr>').addClass('Skater').appendTo(table);
+		$('<td>').attr('colspan', '3')
+			.append($('<input type="text">').attr('size', '35').attr('id', 'annotation'))
+			.append($('<button>').attr('id', 'setAnnotation').text('Set').button().click(function() {
```

Comment:
Do we really need this Set button? I'd expect this to be submitted automatically via an oninput handler.